sbt/src/sbt-test/project/scripted-skip-incompatible/build.sbt

```diff
@@ -0,0 +1,2 @@
+lazy val root = (project in file("."))
+  .enablePlugins(SbtPlugin)
```

Comment:
@dwijnand The Travis failure was legit. I had to migrate to the new way that explicitly enables the use of scripted at subproject definition.